graphics.h

```diff
@@ -0,0 +1,30 @@
+#ifndef GRAPHICS_H
+#define GRAPHICS_H
+
+#include <SDL2/SDL.h>
```

Comment:
We don't need SDL.h in this file. All the SDL specific stuff is hidden in the IMPLEMENTATION or DEFINITION (the .cpp file) rather than the INTERFACE or DECLARATION (the .h file).